internal/builders/golang/build.go

```diff
@@ -119,6 +124,23 @@ func (*Builder) Build(ctx *context.Context, build config.Build, options api.Opti
 	if err := run(ctx, cmd, env, build.Dir); err != nil {
 		return errors.Wrapf(err, "failed to build for %s", options.Target)
 	}
+
+	if build.ModTimestamp != "" {
```

Comment:
in theory this will never be empty, as we sett `{{ .Timestamp }}` by default... 

maybe we should although leave it empty by default (so it has the old behavior) and leave this if here... wdyt?

Reply:
Yeah, I was thinking of using empty string to signify the previous behavior of not updating the times, so probably best to default to that. Will make the changes.

Reply:
Thanks!